src/app/shared/utils/follow-link-config.model.ts

```diff
@@ -28,6 +28,17 @@ export class FollowLinkConfig<R extends HALResource> {
    * Forward to rest which links we're following, so these can already be embedded
    */
   shouldEmbed? = true;
+
+  /**
+   * If this is true, the link will only be retrieved if there's no valid cached version. Defaults
+   * to true
+   */
+  useCachedVersionIfAvailable? = true;
+
+  /**
+   *
+   */
+  reRequestOnStale? = true;
```

Comment:
Please fill out the (currently empty) description above.

Reply:
Done